models.js

```diff
@@ -0,0 +1,40 @@
+const mongoose = require('mongoose');
+
+const user = new mongoose.Schema({
+  first: {
+      type: String,
+      required: true
+  },
+  last: {
+      type: String,
+      required: true
+  },
+  birthday: String,
```

Comment:
This is a little redundant.  I'd make `birthday` a `Date` and remove the `date` property